setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="timewriggler",
-    version="2.0.1",
+    version="2.0.2",
```

Comment:
I think with the change in the way we parse dates; and printing missing projects, this is a more of a feature bump release!
```suggestion
    version="2.1.0",
```